drivers/clock_control/clock_control_it8xxx2.c

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define DT_DRV_COMPAT ite_it8xxx2_ecpm
+
+#include <soc.h>
+#include <zephyr/logging/log.h>
+#include "soc_espi.h"
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/dt-bindings/clock/it8xxx2_clock.h>
+#include <zephyr/drivers/clock_control/it8xxx2_clock_control.h>
+
+LOG_MODULE_REGISTER(clock_control_it8xxx2, LOG_LEVEL_ERR);
```

Comment:
Why not `CONFIG_CLOCK_CONTROL_LOG_LEVEL`?

Reply:
> Why not `CONFIG_CLOCK_CONTROL_LOG_LEVEL`?

@edersondisouza 

Appreciative of your guidance that I forgot to replace it with the CONFIG_CLOCK_CONTROL_LOG_LEVEL, this upload should fix it. 

Thanks again.